contracts/RewardSafeDelegateImplementation.sol

```diff
@@ -0,0 +1,75 @@
+pragma solidity ^0.5.17;
+
+import "@openzeppelin/contract-upgradeable/contracts/token/ERC20/IERC20.sol";
+import "./RewardManager.sol";
+import "./ActionDispatcher.sol";
+import "@gnosis.pm/safe-contracts/contracts/GnosisSafe.sol";
+
+// This contract is used as an implementation for delegatecall usage of safe operations.
+// It will never be called from it's own deployment address, whenever code here executes,
+// address(this) will be the gnosis safe, and if you call a function on another contract,
+// that other contract will see msg.sender as the safe address
```

Comment:
Can we add a note here stating that the `msg.sender` on this contract is the sender of the transaction. In our case the relayer